examples/flax/language-modeling/run_clm_flax.py

```diff
@@ -187,16 +186,16 @@ def data_loader(rng: jax.random.PRNGKey, dataset: Dataset, batch_size: int, shuf
     steps_per_epoch = len(dataset) // batch_size
 
     if shuffle:
```

Comment:
@patil-suraj - I think here we could actually switch to `jax.random.permutation` as this is called only once per epoch. I'm pretty sure that it wouldn't slow down the training